tests/accelerators/test_tpu.py

```diff
@@ -0,0 +1,16 @@
+from pytorch_lightning.accelerators import TPUAccelerator
+from pytorch_lightning.plugins import SingleTPUPlugin
+from pytorch_lightning.plugins.training_type import TPUSpawnPlugin
+from tests.helpers.runif import RunIf
+
+
+@RunIf(tpu=True)
+def test_device_stats_tpu(tmpdir):
```

Comment:
This TPU test won't run as this file needs to be mentioned in the test suite for `tpus`. Reference https://github.com/PyTorchLightning/pytorch-lightning/blob/master/dockers/tpu-tests/tpu_test_cases.jsonnet#L38

Reply:
Ah, thank you I was confused about that!